docs/index.asciidoc

```diff
@@ -141,7 +142,15 @@ E.g: `/%{myfield}/`, `/test-%{myfield}/` are not valid paths
 If `append`, the file will be opened for appending and each new event will be written at the end of the file.
 If `overwrite`, the file will be truncated before writing and only the most recent event will appear in the file.
 
-[id="plugins-{type}s-{plugin}-common-options"]
+[id="plugins-{type}s-{plugin}-stale_cleanup_interval"]
+===== `stale_cleanup_interval`
+
+  * Value type is <<number,number>>
+  * Default value is `10`
+
+Defines the interval, in seconds, between the stale files cleanup runs (in seconds).
```

Comment:
"In seconds" is in the sentence twice. Please remove one instance.  
I like the first one, but you can decide how you want to do it.